### ISSUE TITLE
Add LWP::Protocol::https

### ISSUE THREAD

**Issue by JJ** [actions/runner-images#10567]
### Tool name

LWP::Protocol::https

### Tool license

Artistic

### Add or update?

- [ ] Add
- [X] Update

### Desired version

latest

### Brief description of tool

Runners already include LWP::Simple and LWP::UserAgent, Perl libraries that take care of making HTTP(S) requests. However, they need this separate library to actually make `https` requests.

### URL for tool's homepage

https://metacpan.org/dist/LWP-Protocol-https

### Provide a basic test case to validate the tool's functionality.

```bash
perl -MLWP::Simple -e 'getprint "https://jj.github.io/IV"'
```
```


### Platforms where you need the tool

- [X] Azure DevOps
- [X] GitHub Actions

### Runner images where you need the tool

- [X] Ubuntu 20.04
- [X] Ubuntu 22.04
- [X] Ubuntu 24.04
- [X] macOS 12
- [X] macOS 13
- [X] macOS 13 Arm64
- [X] macOS 14
- [X] macOS 14 Arm64
- [X] Windows Server 2019
- [X] Windows Server 2022

### Can this tool be installed during the build?

```bash
It can be installed in Ubuntu through its package https://launchpad.net/ubuntu/focal/+package/liblwp-protocol-https-perl

There are probably other ways of installing it in Windows/MacOS, but I really have no idea.
```

### Tool installation time in runtime

16 seconds through apt

### Are you willing to submit a PR?

Certainly.

**Comment by hemanthmanga** [comment]
Hi @JJ Thank you for bringing this issue to us. We are looking into this issue and will update you on this issue after investigating.

**Comment by JJ** [comment]
> Hi @JJ Thank you for bringing this issue to us. We are looking into this issue and will update you on this issue after investigating.

As a matter of fact, I've looked up a list of Perl modules installed some time ago, and it was actually there; that should have been Ubuntu-18 or thereabouts.

**Comment by JJ** [comment]
Certainly, as it was there in other versions. I use it to access APIs, or to scrape; in general, it gives support to LWP for the protocol that is used the most. Without it, the runner lacks this functionality and you need a lengthy installation process, a cache or a container
